man/rules/meson.build

```diff
@@ -11,7 +11,6 @@ manpages = [
  ['coredump.conf', '5', ['coredump.conf.d'], 'ENABLE_COREDUMP'],
  ['coredumpctl', '1', [], 'ENABLE_COREDUMP'],
  ['crypttab', '5', [], 'HAVE_LIBCRYPTSETUP'],
- ['veritytab', '5', [], 'HAVE_LIBCRYPTSETUP'],
```

Comment:
Why do people keep editing the file by hand?! There is both a header *and* a footer saying not to do that ;(

Reply:
I thought we were supposed to run update-man-rules? Might be wrong here.

Reply:
Your patch is fine, it's just fixing an issue introduced when veritytab was added a few days ago.